webapp/locales/de.json

```diff
@@ -415,6 +415,14 @@
     "languageSelectLabel": "Sprache",
     "categories": {
       "infoSelectedNoOfMaxCategories": "{chosen} von {max} Kategorien ausgewählt"
+    },
+    "emotions-label": {
+      "funny": "Lustig",
+      "happy": "Glücklich",
+      "surprised": "Erstaunt",
+      "cry": "Zum Weinen",
+      "angry": "Verärgert",
+      "emoted": "emotiert"
```

Comment:
The word `emotiert` just doesn’t exist in german, as far as I know. And I even don’t know anything similar.
I would set the word to `angegeben`.